ssh-dss.c

```diff
@@ -66,6 +66,34 @@ ssh_dss_cleanup(struct sshkey *k)
 	k->dsa = NULL;
 }
 
+static int
+ssh_dss_equal(const struct sshkey *a, const struct sshkey *b)
+{
+	const BIGNUM *dsa_p_a, *dsa_q_a, *dsa_g_a, *dsa_pub_key_a;
```

Comment:
ok to me, but my brain works different and I would have written dsa_a_p and dsa_b_p, etc

Reply:
never mind